lib/traject/config/folio_config.rb

```diff
@@ -251,7 +252,7 @@ def holdings(record, context)
 ##
 # Skip records for missing `item_display` field
 each_record do |_record, context|
-  if context.output_hash['item_display'].blank? && context.output_hash['url_fulltext'].blank?
+  if context.output_hash['item_display'].blank? && context.output_hash['url_fulltext'].blank? && settings['skip_empty_item_display'] > -2
```

Comment:
Is there a reason we're testing on -2 here?  The docs says we'll skip on anything `> -1` https://github.com/sul-dlss/searchworks_traject_indexer/blob/0a4bf72d618f9fd8a28760ae97ab64bdd1af9f99/README.md?plain=1#L170

Reply:
Just for conveniently reviewing this PR separate from #1103 (where it changes back to `-1`: https://github.com/sul-dlss/searchworks_traject_indexer/pull/1103/files#diff-93251714c685d86f273f6f69bd7e74da3770b4c810dd1fc5f1a0720033586578R2361.)

We have two similar `skip` steps in `folio_config` and `marc_reader`, and I'm using -2 to skip the `marc_reader` version but not this one.